vividus-to-azure-devops-exporter/src/main/java/org/vividus/azure/devops/facade/AzureDevOpsFacade.java

```diff
@@ -281,8 +281,12 @@ private AddOperation createDescriptionOperation(String value)
     private List<AddOperation> createCommonOperations(String testTitle)
     {
         List<AddOperation> operations = new ArrayList<>();
-        operations.add(new AddOperation("/fields/System.AreaPath", format("%s\\%s", options.getProject(),
-                options.getArea())));
+        String areaPath = options.getProject();
+        if (options.getArea() != null)
```

Comment:
```java
private Optional<String> area;
```
?

Reply:
Spring won't set optional empty into this